Test

```diff
@@ -0,0 +1,1012 @@
+===============================================================================
+=    W i t a j   w   t u t o r i a l u   V I M - a      -    Wersja  1.7.     =
+===============================================================================
+<<<<<<< HEAD
+
+=======
+ 
+>>>>>>> vimtutor
```

Comment:
Your merge was not successful. Please fix it. If you need help let us know on Discord.
+ the name of this file should be tutor.